src/components/DocumentProducer/MultiLineComponentNonEditable/index.tsx

```diff
@@ -19,23 +43,158 @@ export default function MultiLineComponentNonEditable({
   title,
   description,
   status,
+  variableId,
+  projectId,
+  reload,
+  ownerId,
 }: MultiLineComponentNonEditableProps): JSX.Element {
+  const { activeLocale } = useLocalization();
+  const dispatch = useAppDispatch();
+  const snackbar = useSnackbar();
+  const [requestId, setRequestId] = useState('');
+  const [assignOwnerRequestId, setAssignOwnerRequestId] = useState('');
+  const updateWorkflowDetailsResponse = useAppSelector(selectUpdateVariableWorkflowDetails(requestId));
+  const updateOwnerResponse = useAppSelector(selectUpdateVariableOwner(assignOwnerRequestId));
+  const ownedBySelector = useAppSelector(selectUser(ownerId));
+  const [ownedByUser, setOwnedByUser] = useState<User>();
+  const [showAssignOwnerModal, setShowAssignOwnerModal] = useState(false);
+  const [displayActionsHover, setDisplyActionsHover] = useState(false);
+
+  useEffect(() => {
+    setOwnedByUser(ownedBySelector);
+  }, [ownedBySelector]);
+
+  useEffect(() => {
+    if (ownerId && ownerId !== -1) {
+      dispatch(requestGetUser(ownerId));
+    }
+  }, [ownerId]);
+
+  useEffect(() => {
+    if (updateWorkflowDetailsResponse?.status === 'success') {
+      reload();
+    }
+  }, [updateWorkflowDetailsResponse]);
+
+  useEffect(() => {
+    if (updateOwnerResponse?.status === 'success') {
+      snackbar.toastSuccess(strings.SECTION_OWNER_ASSIGNED);
+      reload();
+    }
+  }, [updateOwnerResponse]);
```

Comment:
Should we provide feedback for these requests when the response is unsuccessful?

Reply:
Yes, probably show the generic error. Will update it